redaxo/src/core/pages/setup.php

```diff
@@ -157,22 +161,22 @@
     $fragment->setVar('buttons', $buttons, false);
     echo '<div class="rex-js-setup-section">' . $fragment->parse('core/page/section.php') . '</div>';
     echo $security;
+
+    return;
 }
 
 // ---------------------------------- step 4 . Config
 
 $error_array = [];
 
-if ($step >= 4) {
-    $configFile = rex_path::coreData('config.yml');
-    $config = array_merge(
-        rex_file::getConfig(rex_path::core('default.config.yml')),
-        rex_file::getConfig($configFile)
-    );
+$configFile = rex_path::coreData('config.yml');
+$config = array_merge(
+    rex_file::getConfig(rex_path::core('default.config.yml')),
+    rex_file::getConfig($configFile)
+);
```

Comment:
Hier ist der Block, um den es vor allem in #2885 ging. Ist nun also in der obersten Ebene und das Ursprungsproblem sollte somit auch behoben sein.